### ISSUE TITLE
Why are there no new images past 2022-07-18 ?

### ISSUE THREAD

**Issue by AaronNGray** [Seeed-Studio/seeed-linux-openwrt#35]
Why are there no new images past 2022-07-18 ?

Has there not been an OpenSSL/SSH vulnerability since then ?

**Comment by charliebearsson** [comment]
Because they are NOT looking out for their consumers. Do like I did.

I just installed official openwrt with plain luci. Everything worked. Just had to set up DHCP interface for WAN under interfaces.
Flashed this version with balenaEtcher.
How to flash: https://wiki.seeedstudio.com/OpenWrt-Getting-Started/
Firmware: https://openwrt.org/toh/hwdata/raspberry_pi_foundation/raspberry_pi_foundation_raspberry_pi_cm4

Works good so far, will see in a couple of days how the throughput works. I want to use this thing as a router and nothing else.

**Comment by AaronNGray** [comment]
Thanks, I think using OpenWRT image with a package update with :- (https://github.com/Seeed-Studio/seeed-linux-openwrt/blob/magic/scripts/update_packages.py) will do the job.
The seeeed stuff runs ontop of OpenWRT.

**Comment by charliebearsson** [comment]
> Thanks, I think using OpenWRT image with a package update with :- (https://github.com/Seeed-Studio/seeed-linux-openwrt/blob/magic/scripts/update_packages.py) will do the job. The seeeed stuff runs ontop of OpenWRT.

Okay, will probably work. I'm experiencing problems with throughput and a bunch of other stuff so I want everything SEEED gone. I know it runs on top, and I personally use none of it. I want a fast router with support of wireguard.

**Comment by AaronNGray** [comment]
@charliebearsson You just flashed the squashfs ?

**Comment by charliebearsson** [comment]
If you have the rpi4 mini seeed router you download this firmware: https://downloads.openwrt.org/releases/22.03.2/targets/bcm27xx/bcm2711/openwrt-22.03.2-bcm27xx-bcm2711-rpi-4-squashfs-factory.img.gz
and follow this: https://wiki.seeedstudio.com/OpenWrt-Getting-Started/
It will tell you exactly how to flash this image. The only thing that you should do is switch to the image i linked to in this post.
Then plug the ethernet cables back into it. The NIC port closest to the USB-C power port is WAN and the otherone is LAN. 

login in with browser. (mine is 192.168.1.1), no password at first boot. just user: root. password is changed in luci. 
Go to Network > interfaces. 
Press edit on lan interface, set device to eth0, save and apply.
Add interface, name it WAN, protocol: DHCP, device: eth1, check bring back on boot. Save and apply
Add interface, name it WAN6, protocol: DHCPv6, device: eth1, check bring back on boot. Save and apply

Dont forget to change the password for your router under system > administration

Good luck, check back if you need any further help. I'm no pro but I got it working....

**Comment by AaronNGray** [comment]
@charliebearsson  Hi, Still woring alright ? If so I will finally get round to reflashing the Seeed reRouter with OpenWRT this week !

**Comment by kurzninja** [comment]
I just installed vanilla OpenWRT tonight and after some fumbling and factory restoring to finally get internet back, it seems like I'm up and running without the SEEED Studios broken firmware. And my SpeedTest.net results went from around 70-80 Mbps to now being all the way up to close to 300 Mbps. What a difference. Hopefully I don't have to reboot my router every day anymore just to maintain performance. Thanks a lot @charliebearsson !

**Comment by kurzninja** [comment]
Been over a week now and speed hasn't changed since I flashed the OpenWRT. Still getting upwards of 300mbps speed without needing a reboot. The changeover was a piece of cake and the initial setup was not without its bumps (for some reason, setting up the WAN didn't work the first time until I did a hard reset in the software and started over, then it worked). But all in all, pretty easy and worth it. Not sure what the hell Seeed Studio's firmware was doing to have these issues and such crappy performance in the first place.
